arch/x86/pagetables.c

```diff
@@ -146,7 +146,11 @@ static mfn_t get_pgentry_mfn(mfn_t tab_mfn, pt_index_t index, unsigned long flag
     return mfn;
 }
 
-void *vmap(void *va, mfn_t mfn, unsigned int order, unsigned long flags) {
+void *vmap_ext(void *va, mfn_t mfn, unsigned int order,
```

Comment:
I think this is intended to be an internal function? If so, please classify it as static and rename to `_vmap`.